client/src/components/ProfilePhoto/useStyles.ts

```diff
@@ -0,0 +1,60 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles(() => ({
+  root: {
+    marginTop: '10rem',
+    marginBottom: '10rem',
+    alignItems: 'center',
+    justifyContent: 'center',
+    minWidth: '20rem',
+  },
+  profileContainer: {
+    alignItems: 'center',
+    minHeight: '60vh',
+    margin: 'auto',
+  },
+  contentWrapper: {
+    display: 'flex',
+    justifyContent: 'center',
+  },
+  title: {
+    marginTop: '5rem',
+  },
+  titleText: {
+    fontFamily: 'Roboto',
+    fontWeight: 'bolder',
+    fontSize: '1.5rem',
+    margin: 'auto',
+  },
+  avatar: {
+    marginTop: '4rem',
+    width: 150,
+    height: 150,
+  },
+  caption: {
+    marginTop: '2rem',
+    color: '#9e9e9e',
+    width: '10rem',
+    textAlign: 'center',
```

Comment:
You can pass this to the `Typography` component instead, using its [`align` prop](https://v4.mui.com/api/typography/#props).